fv3core/grid/eta.py

```diff
@@ -0,0 +1,183 @@
+                6698.281,
+                6290.007,
+                5884.984,
+                5484.372,
+                5089.319,
+                4700.96,
+                4320.421,
+                3948.807,
+                3587.201,
+                3236.666,
+                2898.237,
+                2572.912,
+                2261.667,
+                1965.424,
+                1685.079,
+                1421.479,
+                1175.419,
+                947.6516,
+                738.8688,
+                549.713,
+                380.7626,
+                232.5417,
+                105.481,
+                -0.0008381903,
+                0,
+            ]
+        )
+        bk = np.array(
+            [
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0.001065947,
+                0.004128662,
+                0.009006631,
+                0.01554263,
+                0.02359921,
+                0.03305481,
+                0.0438012,
+                0.05574095,
+                0.06878554,
+                0.08285347,
+                0.09786981,
+                0.1137643,
+                0.130471,
+                0.1479275,
+                0.1660746,
+                0.1848558,
+                0.2042166,
+                0.2241053,
+                0.2444716,
+                0.2652672,
+                0.286445,
+                0.3079604,
+                0.3297701,
+                0.351832,
+                0.3741062,
+                0.3965532,
+                0.4191364,
+                0.4418194,
+                0.4645682,
+                0.48735,
+                0.5101338,
+                0.5328897,
+                0.5555894,
+                0.5782067,
+                0.6007158,
+                0.6230936,
+                0.6452944,
+                0.6672683,
+                0.6889648,
+                0.7103333,
+                0.7313231,
+                0.7518838,
+                0.7719651,
+                0.7915173,
+                0.8104913,
+                0.828839,
+                0.846513,
+                0.8634676,
+                0.8796583,
+                0.8950421,
+                0.9095779,
+                0.9232264,
+                0.9359506,
+                0.9477157,
+                0.9584892,
+                0.9682413,
+                0.9769447,
+                0.9845753,
+                0.9911126,
+                0.9965372,
+                1,
+            ]
+        )
+        ks = 18
+    else:
+        raise NotImplementedError(
+            "Only grids with 79 Vertical Levels have been implemented so far"
```

Comment:
We support specifically one 79-level grid, not multiple. Keep in mind this file shouldn't contain context about horizontal grids.
```suggestion
            "Only a 79 vertical level grid has been implemented so far"
```